### PR TITLE
fix: error handling in CLI and templates documentation

### PR DESCRIPTION
**Pull request by vdusek**
### Description

- [x] Rename `local_storage_dir` to `storage_dir`.
- [x] Mention templates and CLI in readme.
- [x] Mention templates and CLI in doc 
- [x] Fix CLI when project directory already exists.
- [x] Handle CTRL+C in CLI.
- ~~Poetry templates to Pip templates.~~

### Issues

- Related #266
- Closes: #268

### Testing

- All scenarios were executed locally. Handling CTRL+C is still not perfect. However, this should be ok for the initial release.

### Checklist

- [x] Changes are described in the `CHANGELOG.md`
- [x] CI passed

### CODE REVIEW

File: src/crawlee/configuration.py [hunk 1]
```diff
@@ -166,12 +166,12 @@ class Configuration(BaseSettings):
...
     ] = None
 
-    local_storage_dir: Annotated[
+    storage_dir: Annotated[
         str,
         Field(
             validation_alias=AliasChoices(
-                'apify_local_storage_dir',
-                'crawlee_local_storage_dir',
+                'apify_storage_dir',
```
Comment (B4nan) [inline src/crawlee/configuration.py:173]: this one should be still `apify_local_storage_dir` i think

File: tests/unit/memory_storage_client/test_memory_storage_client.py [hunk 2]
```diff
@@ -21,13 +21,13 @@ async def test_write_metadata(tmp_path: Path) -> None:
     dataset_no_metadata_name = 'test-no-metadata'
     ms = MemoryStorageClient(
         Configuration(
-            crawlee_local_storage_dir=str(tmp_path),  # type: ignore
+            crawlee_storage_dir=str(tmp_path),  # type: ignore
```
Comment (B4nan) [inline tests/unit/memory_storage_client/test_memory_storage_client.py:24]: same here

File: tests/unit/conftest.py [hunk 3]
```diff
@@ -43,15 +43,15 @@ def _isolate_test_environment(monkeypatch: pytest.MonkeyPatch, tmp_path: Path) -
...
+    assert os.environ.get('CRAWLEE_STORAGE_DIR') == str(tmp_path)
 
 
 @pytest.fixture()
 def memory_storage_client(tmp_path: Path) -> MemoryStorageClient:
     cfg = Configuration(
         write_metadata=True,
         persist_storage=True,
-        crawlee_local_storage_dir=str(tmp_path),  # type: ignore
+        crawlee_storage_dir=str(tmp_path),  # type: ignore
```
[the remark this answers is not in the document]
Comment (vdusek) [reply tests/unit/conftest.py:54]: This is just some assignment/override thing and it has reasons. See the https://github.com/apify/crawlee-python/issues/146. I will add a comment to this file as well. It's thanks to the Pydantic bug https://github.com/pydantic/pydantic-settings/issues/180. However, the field is called `local_storage_dir` (currently), check the API docs - https://crawlee.dev/python/api/class/Configuration#local_storage_dir.

File: tests/unit/memory_storage_client/test_memory_storage_client.py (hunk 2 above)
Comment (vdusek) [reply tests/unit/memory_storage_client/test_memory_storage_client.py:24]: same answer

File: tests/unit/conftest.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (B4nan) [reply tests/unit/conftest.py:57]: Yes, and that is wrong, it should be called just `storage_dir`. The concept of "local storage dir" is only about the Apify platform, where "remote" means on platform. For those who do not use the platform it's just a storage dir, regardless of where they run it.

[the remark this answers is not in the document]
Comment (vdusek) [reply tests/unit/conftest.py:57]: yes, the field is called `storage_dir` in this PR

**Pull request merged**